python/lsst/ts/scheduler/driver/driver_target.py

```diff
@@ -189,7 +190,9 @@ def get_script_config(self) -> dict:
         script_config = {
             "targetid": int(self.targetid),
             "band_filter": str(self.filter),
+            "filter_name": self.get_filter_name(),
             "name": self.get_target_name(),
+            "note": str(self.note),
```

Comment:
It looks like `note` already is a string. Why is an explicit cast necessary here?

Reply:
This is tied to the issue you pointed later about the necessity of casting `float` for `rot` and `rot_sky`. In the feature scheduler these things are numpy types (`np.string` and `np.float`), which won't serialize correctly to json/yaml later. So, to avoid serialization issues we need to cast them here.